web/nabs.yml

```diff
@@ -0,0 +1,28 @@
+# -- BUILD
```

Comment:
nabs?

Reply:
Handy NPM utility for writing `package.json` scripts without the usual pain that comes with doing that.